src/axom/primal/operators/compute_moments.hpp

```diff
@@ -103,6 +109,9 @@ T area(const primal::CurvedPolygon<T, 2>& poly, double tol = 1e-8)
   {
     for(int ed = 0; ed < ngon; ++ed)
     {
+      // Algorithm works only on nonrational Bezier curves
+      SLIC_ASSERT(!ed.isRational());
```

Comment:
Typo:
Looks like this should be:
```suggestion
      SLIC_ASSERT(!poly[ed].isRational());
```